src/api/objects/ObjectAPI.js

```diff
@@ -298,10 +298,13 @@ ObjectAPI.prototype.save = function (domainObject) {
                 savedResolve = resolve;
             });
             domainObject.persisted = persistedTime;
-            provider.create(domainObject).then((response) => {
-                this.mutate(domainObject, 'persisted', persistedTime);
-                savedResolve(response);
-            });
+            const newObject = provider.create(domainObject);
+            if (newObject) {
```

Comment:
I'm pretty sure we don't want to just suppress this error if it happens. I think we want to reject the promise on line 297.